containers/unit_tests/TestBitset.hpp

```diff
@@ -185,8 +185,8 @@ void test_bitset() {
   typedef Kokkos::Bitset<Device> bitset_type;
   typedef Kokkos::ConstBitset<Device> const_bitset_type;
 
-  std::array<unsigned, 7> test_sizes = {0u,       10u,      100u,    1000u,
-                                        1u << 14, 1u << 16, 10000001};
+  std::array<unsigned, 7> test_sizes = {{0u,       10u,      100u,    1000u,
+                                        1u << 14, 1u << 16, 10000001}};
```

Comment:
```suggestion
  std::array<unsigned, 7> test_sizes = {
      {0u, 10u, 100u, 1000u, 1u << 14, 1u << 16, 10000001}};
```

Reply:
Fixed the formatting and pushed rebased commit